Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM alpine:3.5
+
+COPY gopath/bin/external-dns /external-dns
```

Comment:
I think `gopath/bin` will make it a bit inconvenient if you want to build a Dockerfile locally.

I would expect it copies from `./` or from `build/` (if this was defined in the Makefile).